### PR TITLE
Add ability to generate mash scripts

### PR DESCRIPTION
**Pull request by jlsherrill**
and check in3.3 mash scripts

### CODE REVIEW

File: lib/tool_belt/mash_script.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_path = source/SRPMS
+debuginfo = False
+multilib = False
+multilib_method = devel
+tag = #{self.tag}
+inherit = True
+strict_keys = False
+keys = #{self.gpgkey}
+arches = x86_64
+      EOF
```
Comment (ehelms) [inline lib/tool_belt/mash_script.rb]: Thoughts on moving this to a stored file and using something like ERB templating?

Comment (jlsherrill) [inline lib/tool_belt/mash_script.rb]: Don't really care either way.  Its making it a bit more complex but nicer for development I guess.  I'd say right now I slightly lean against doing it, if we add a second instance of something similar I'd probably be completely in favor of it.  But i'm okay either way

**Comment by ehelms** [comment]
Looks good, the one general comment. Also, this only generates the mash configs right? Not the actual script we run to mash?

File: lib/tool_belt/mash_script.rb (hunk 1 above)
Comment (ehelms) [inline lib/tool_belt/mash_script.rb]: Hrmm, at first I thought this was generating them and deploying them but the idea is to use the CLI to generate them and then commit them here?

**Comment by ehelms** [comment]
@jlsherrill bump

**Comment by jlsherrill** [comment]
Correct, we could add something maybe but we might want to restructure most
of the script to be reusable if we go that route

**Comment by jlsherrill** [comment]
That was thinking, still requires to be copied to /etc/mash

**Comment by ehelms** [comment]
ACK - thanks @jlsherrill

**Pull request merged**